### PR TITLE
Put default memory to mem spec

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: mkdocs/docs/HPC/fine_tuning_job_specifications.md [hunk 1]
```diff
@@ -295,6 +295,15 @@ or on the command line
 qsub -l mem=4gb
 ```
 
+{% if site==gent %}
+If you do not request memory (neither in the job script nor on the command line), the default memory
+that your job will get access is the proportional share of the total memory on the node:
```
Comment (stdweird) [inline mkdocs/docs/HPC/fine_tuning_job_specifications.md:300]: total memory -> total usable memory

File: mkdocs/docs/HPC/fine_tuning_job_specifications.md [hunk 2]
```diff
@@ -295,6 +295,15 @@ or on the command line
...
+If you request a full node, all memory will be available.
```
Comment (stdweird) [inline mkdocs/docs/HPC/fine_tuning_job_specifications.md:301]: all -> all usable

File: mkdocs/docs/HPC/fine_tuning_job_specifications.md [hunk 3]
```diff
@@ -295,6 +295,15 @@ or on the command line
...
+If you request `N` cores on a cluster where nodes have `M` cores, you will get `N/M`
+of the total memory on the node. For the number of cores and available memory per cluster, please see our
+[infrastructure](https://www.ugent.be/hpc/en/infrastructure).
```
Comment (stdweird) [inline mkdocs/docs/HPC/fine_tuning_job_specifications.md:304]: or go to the portl, open desktop app and there you can browse it per cluster and core

File: mkdocs/docs/HPC/fine_tuning_job_specifications.md [hunk 4]
```diff
@@ -295,6 +295,15 @@ or on the command line
 qsub -l mem=4gb
 ```
 
+{% if site==gent %}
+If you do not request memory (neither in the job script nor on the command line), the default memory
+that your job will get access is the proportional share of the total memory on the node:
+If you request a full node, all memory will be available.
+If you request `N` cores on a cluster where nodes have `M` cores, you will get `N/M`
+of the total memory on the node. For the number of cores and available memory per cluster, please see our
```
Comment (itkovian) [inline mkdocs/docs/HPC/fine_tuning_job_specifications.md:303]:
```suggestion
of the total usable memory on the node. For the number of cores and available memory per cluster, please see our
```

File: mkdocs/docs/HPC/fine_tuning_job_specifications.md [hunk 5]
```diff
@@ -297,11 +297,12 @@ qsub -l mem=4gb
...
-that your job will get access is the proportional share of the total memory on the node:
-If you request a full node, all memory will be available.
+that your job will get access is the proportional share of the total avaliable memory on the node:
+If you request a full node, all usable memory will be available.
 If you request `N` cores on a cluster where nodes have `M` cores, you will get `N/M`
-of the total memory on the node. For the number of cores and available memory per cluster, please see our
-[infrastructure](https://www.ugent.be/hpc/en/infrastructure).
+of the total usable memory on the node. For the number of cores and available memory per cluster, please see our
+[infrastructure](https://www.ugent.be/hpc/en/infrastructure), or you can use the [web portal](./web_portal.md), open
+the desktop app and there you can browse it per cluster and core.
```
Comment (stdweird) [inline mkdocs/docs/HPC/fine_tuning_job_specifications.md:305]: `per cluster and core` -> `per cluster and core using the submission form (there is no need to start an actual desktop)`

**Pull request merged**